requirements.txt

```diff
@@ -1,10 +1,13 @@
+cx_Oracle >= 7.1.0
 numpy>1.7,<1.12; python_version == '2.6'
 numpy>1.7,<1.12; python_version == '3.2'
 numpy>1.7,<1.12; python_version == '3.3'
 numpy>1.7,<1.15; python_version == '2.7'
 numpy>1.7; python_version >= '3.4'
+pandas >= 0.24.1
 ## currently conflicts if numpy isn't installed, as it will download it's own version due to steup_requries
 root_numpy
+sqlalchemy >= 1.2.17
```

Comment:
is this specific version needed?
`cc7` available is `0.9.8-2.el7`

Reply:
I’ll try with that version

Reply:
Available version in OS works.  So I should remove the requirement or specify `>=` the version available in cc7?